src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -5,7 +5,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Comment:
With .NET 5 being out of support would we want to use `net6.0` here? https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

Reply:
This is just a library; the downstream project determines the actual runtime. Updating to net6.0 doesn't buy us anything here.

Reply:
When we publish NuGet pkg for this project, it will be .Net 5, which is no longer supported.

Reply:
The .NET policy is to drop TFs that are no longer in support. This is adding one which is already out of support. We should not be doing this. 

@terrajobst